elastic/kibana
# [Security Solution] Introduce method to clean up/delete Risk Scoring resources

## Summary

Create a method that deletes all resources created by the risk engine. The method returns a list of errors and an empty array for the success scenario.

Signature:
```js
const errors = await riskEngineDataClient.tearDown({
  taskManager,
  riskScoreDataClient,
});
```

# How to test it?
1. Enable the risk engine
2. Call the method
3. It should return an empty array

-----
1. On a clean instance
2. Call the method
4. It should return a list of errors

### Checklist
